qcelemental/molparse/to_string.py

```diff
@@ -152,13 +152,27 @@ def to_string(molrec: Dict,
 
         atoms = _atoms_formatter(molrec, geom, atom_format, ghost_format, width, prec, 2)
 
-        first_line = f"""{{{umap.get(units.lower())}}}"""
-        second_line = """geometry={"""
+        smol = []
+
+        # Don't orient the molecule if asked to fix_com or fix_orientation
+        if molrec['fix_orientation'] or molrec['fix_com']:
+            smol.append('{orient,noorient}')
+
+        # Have no symmetry if asked to fix_symmetry
+        if 'fix_symmetry' not in molrec.keys():
+            smol.append('{symmetry,nosym}')
```

Comment:
sorry, I must have commented vaguely. _Two_ cases, I think: (a) 'fix_symmetry' present and == 'c1' --> `{symmetry,nosym}`, (b) 'fix_symmetry' absent --> `{symmetry,auto}`.

Reply:
Oh sorry, I misunderstood the previous comment. I've updated the code to handle the two cases.

Reply:
Oh and to clarify did you want the case for 'fix_symmetry' present and !='c1' to be given nothing right now?

Reply:
Yeah, nothing to do for that case I think. There was sym=cs bellow but that’s for thermo, not for molecule.